streampark-console/streampark-console-webapp/src/locales/lang/zh-CN/flink/app.ts

```diff
@@ -62,7 +62,7 @@ export default {
   programJar: '程序Jar文件',
   dynamicProperties: '动态参数',
   hadoopConfigTitle: '系统 Hadoop',
-  dragUploadTitle: '单击或拖动 jar 到此区域以上传',
+  dragUploadTitle: '单击或拖动 jar或py 到此区域以上传',
```

Comment:
It is best to modify the English internationalization file synchronously (lang/en/flink/app.ts)

Reply:
Ok, thank you for reviewing the code